catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -90,48 +90,56 @@
   private static final Map<com.google.common.net.MediaType, String>
       FALLBACK_MIME_TYPE_DATA_TYPE_MAP;
 
-  private static final String OVERALL_FALLBACK_DATA_TYPE = "Dataset";
+  private static final String OVERALL_FALLBACK_DATA_TYPE = DataType.DATASET.toString();
 
   static {
     SPECIFIC_MIME_TYPE_DATA_TYPE_MAP = new HashMap<>();
     SPECIFIC_MIME_TYPE_DATA_TYPE_MAP.put(
-        com.google.common.net.MediaType.MICROSOFT_EXCEL, "Document");
+        com.google.common.net.MediaType.MICROSOFT_EXCEL, DataType.TEXT.toString());
```

Comment:
❓`DataType.TEXT` is `Text`, not `Document`. That's what we want, right?

Reply:
We were replacing "Document" with "Text" because that is the closest DCMI type. The only other one that might make sense in this case is Dataset.